### ISSUE TITLE
[DOC] SlidingWindowSplitter

### ISSUE THREAD

**Issue by aiwalter** [alan-turing-institute/sktime#544]
#### Describe the issue linked to the documentation
Additional docstring here would be great to better understand how to configure this:

```python
class SlidingWindowSplitter(BaseWindowSplitter):
    """Sliding window splitter
    Parameters
    ----------
    fh : int, list or np.array
        Forecasting horizon
    window_length : int
    step_length : int
    initial_window : int
    start_with_window : bool, optional (default=True)
    """
```

**Comment by mloning** [comment]
This is related to #477

**Comment by mloning** [comment]
Closed by #555
